rest-api/dao/participant_dao.py

```diff
@@ -127,6 +144,65 @@ def _do_update(self, session, obj, existing_obj):
     self._update_history(session, obj, existing_obj)
     super(ParticipantDao, self)._do_update(session, obj, existing_obj)
 
+
+  def check_for_valid_children(self, session, obj, existing_obj):
+    organization = obj.organizationID
+    site = obj.siteId
+    awardee = obj.hpoId
+    old_site = existing_obj.siteId
+    old_org = existing_obj.organizationID
+    old_awardee = existing_obj.hpoId
+    existing_obj_list = [old_site, old_org, old_awardee]
+    # TODO: DO WE WANT TO PREVENT PAIRING IF EXISTING SITE HAS PM/BIO.
+    #PM = existing_obj.participantSummary.physicalMeasurementsCreatedSiteId
+    #BIO = existing_obj.participantSummary.biospecimenCollectedSiteId
+
+    if site is not None and site != old_site:
+      obj_parent_organization = session.query(Site.organizationId).filter_by(siteId=site).first()
```

Comment:
You can instead just do OrganizationDao().get(Site.organizationId) and then compare... that will use our cache, whereas this query call won't.

We could alternatively add parent fields to Organization and Site. Not sure if that will cause any issues.

Or you could just compare IDs? You don't have to fetch the objects themselves...